File: Bubble_Sort/Bubble_Sort.c
#include <stdio.h>

int* bubble_Sort(int *array, int size)
{
    int i = 1, swap = 1, temp;

    while (swap == 1 && i < size-1)
    {
        swap = 0;

        for (int j = 0; j < size - i - 1; j++)
        {
            if (array[j] > array[j+1])
            {
                temp = array[j];
                array[j] = array[j+1];
                array[j+1] = temp;

                swap = 1;
            }
        }
        i++;
    }
    return array;
}

File: Bubble_Sort/Bubble_Sort.py
def bubble_sort(array):
    swap = 1
    i = 0
    while swap == 1 and i < (len(array)-1):
        swap = 0
        for j in range(i, len(array)-1):
            if array[j] > array[j+1]:
                swap = 1
                array[j], array[j+1] = array[j+1], array[j]
